docs/01-app/03-api-reference/04-functions/revalidatePath.mdx

```diff
@@ -35,7 +35,17 @@ The path parameter can point to pages, layouts, or route handlers:
 
 - **Pages**: Revalidates the specific page
 - **Layouts**: Revalidates the layout and all pages beneath it
-- **GET Route Handlers**: Only if they're statically generated
+- **Route Handlers**: Revalidates Data Cache entries accessed within route handlers. For example `revalidatePath("/api/data")` revalidates this GET handler:
```

Comment:
When I tested this I had `force-static` but also `revalidate = 0` which is 💀 cuz it makes it looks like an statically generated GET handler is revalidating -